src/frontend/src/lib/types/list-params.context.ts

```diff
@@ -0,0 +1,21 @@
+import type { Writable } from 'svelte/store';
+import type { ListFilter, ListOrder, ListParams } from './list';
+
+export enum LIST_PARAMS_KEY {
```

Comment:
For consistency reasons, I'll probably rename it to `ListParamsKey` and use lowercase for the values. Looks good, just mentionning it.